src/graphdb.js

```diff
@@ -127,11 +131,15 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
         });
 
         // Add edge types        
-        edgeTypes.forEach(edgeType => {
+        edgeTypes.forEach(edgeType => {            
```

Comment:
This looks like new logic that we should test. It seems to be part of the logic which is generating the graphQL schema from a graph db schema when a graph db node property is the same as an edge label.